### ISSUE TITLE
Remove confirmation from sanitize_usde_schools

### ISSUE THREAD

**Issue by btbonval** [FinalsClub/karmaworld#284]
There are too many schools output for anyone to make a rational decision about the cleanup to answer "Yes, remove all those schools" or "No, 1 of the 3034 schools to be removed is important."

If a school gets deleted by accident, it's easy enough to draw them all back in with https://github.com/FinalsClub/karmaworld/blob/dab4a237910ee521e5889c31925957e5660033f6/karmaworld/apps/courses/management/commands/import_usde_csv.py

Basically, remove the following highlighted code and adjust indent.
https://github.com/FinalsClub/karmaworld/blob/268bcf89b20b0971ee08be14c013b000e282a321/karmaworld/apps/courses/management/commands/sanitize_usde_schools.py#L67-L72
